src/coreclr/src/System.Private.CoreLib/src/System/Reflection/Emit/MethodBuilder.cs

```diff
@@ -135,14 +135,14 @@ public sealed class MethodBuilder : MethodInfo
 
         #region Internal Members
 
-        internal void CheckContext(params Type[]?[]? typess)
+        internal static void CheckContext(params Type[]?[]? typess)
         {
-            m_module.CheckContext(typess);
+            ModuleBuilder.CheckContext(typess);
         }
```

Comment:
Does this method provide any benefit at this point?  i.e. can callers just use ModuleBuilder.CheckContext directly? (Same goes for the other cases of this below.)